x-pack/test/api_integration/apis/security_solution/timeline_details.ts

```diff
@@ -7,9 +7,12 @@
 import expect from '@kbn/expect';
 import { sortBy } from 'lodash';
 
+// @ts-expect-error
 import { timelineDetailsQuery } from '../../../../plugins/security_solution/public/timelines/containers/details/index.gql_query';
 import {
+  // @ts-expect-error
   DetailItem,
```

Comment:
seems both DetailItem and GetTimelineDetailsQuery are removed from `x-pack/plugins/security_solution/public/graphql/types.ts`